service/controller/resource/scrapeconfigs/resource.go

```diff
@@ -106,13 +124,13 @@ func toSecret(v interface{}, config Config) (*corev1.Secret, error) {
 	return scrapeConfigsSecret, nil
 }
 
-func toData(v interface{}, config Config) ([]byte, error) {
+func toData(v interface{}, config Config, workloadClusterETCDDomain string) ([]byte, error) {
```

Comment:
Please add `workloadClusterETCDDomain` to `config`, I don't see why we need an extra argument here.

Reply:
done